### PR TITLE
Apply PlaySource updates to models and methods

### PR DESCRIPTION
**Pull request by atazimsft**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/PlayOptions.java [hunk 1]
```diff
@@ -33,21 +33,21 @@ public final class PlayOptions {
 
     /**
      * Constructor
-     * @param playSource A {@link PlaySource} representing the source to play.
+     * @param playSources A List of {@link PlaySource} representing the sources to play.
      * @param playTo The targets to play to.
      */
-    public PlayOptions(PlaySource playSource, List<CommunicationIdentifier> playTo) {
-        this.playSource = playSource;
+    public PlayOptions(List<PlaySource> playSources, List<CommunicationIdentifier> playTo) {
```
Comment (wangrui-msft) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/PlayOptions.java:39]: need to have an overload function to accept one playsource object

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/PlayToAllOptions.java [hunk 2]
```diff
@@ -25,19 +27,19 @@ public final class PlayToAllOptions {
 
     /**
      * Constructor
-     * @param playSource A {@link PlaySource} representing the source to play.
+     * @param playSources A List of {@link PlaySource} representing the sources to play.
```
Comment (wangrui-msft) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/PlayToAllOptions.java:30]: overload need for single playsource object input

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java [hunk 3]
```diff
@@ -69,29 +70,29 @@ public final class CallMediaAsync {
...
+     * @param playSources A List of {@link PlaySource} representing the sources to play.
      * @param playTo the targets to play to
      * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return Void for successful play request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Void> play(PlaySource playSource, List<CommunicationIdentifier> playTo) {
-        PlayOptions options = new PlayOptions(playSource, playTo);
+    public Mono<Void> play(List<PlaySource> playSources, List<CommunicationIdentifier> playTo) {
```
Comment (wangrui-msft) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java:80]: need an overload for single playsource input

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/PlayRequest.java [hunk 4]
```diff
@@ -37,22 +37,22 @@ public final class PlayRequest {
...
-     * Set the playSourceInfo property: The source of the audio to be played.
+     * Set the playSources property: The source of the audio to be played.
      *
-     * @param playSourceInfo the playSourceInfo value to set.
+     * @param playSources the playSources value to set.
      * @return the PlayRequest object itself.
      */
-    public PlayRequest setPlaySourceInfo(PlaySourceInternal playSourceInfo) {
-        this.playSourceInfo = playSourceInfo;
+    public PlayRequest setPlaySources(List<PlaySourceInternal> playSources) {
```
Comment (Paresh-Arvind-Patil) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/PlayRequest.java:54]: @atazimsft can you please point me to the overload method which takes play source as a object ?

Comment (Paresh-Arvind-Patil) [reply sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/PlayRequest.java:54]: For your reference we need to have a overload method as we do in C# as below:
image

**Review by Paresh-Arvind-Patil — APPROVED** [review]

**Review by bharat-kalyan-namburi — APPROVED** [review]

**Pull request merged**